devices/pine64-pinephone/u-boot/default.nix

```diff
@@ -58,29 +58,13 @@ in
     # https://patchwork.ozlabs.org/patch/1202024
     (pw "1202024" "0c196zk1s3pq3wdv909sxmjgqpll2hwb817bpbghkfkyyknl96vg")
 
-    # sunxi: support asymmetric dual rank DRAM on A64/R40
-    # https://patchwork.ozlabs.org/patch/1312857
-    (pw "1312857" "0w8caf936y7f1r2lvdqar9n3dfkm461lm3k5hfwxd7biwyd386nk")
-
-    # sunxi: Add arm64 FEL support
-    # https://patchwork.ozlabs.org/patch/1402908
-    (pw "1402908" "1pm6zr7xgafw04rb95237q1mxb8xa6k2lmhzfsj0x0gxal96rw2v")
-
-    # sunxi: dram: h6: Improve DDR3 config detection
-    # https://patchwork.ozlabs.org/patch/1410521
-    (pw "1410521" "1vmalpf3wl3ii592qsh57g11qfrkmx40hlg8mza1330np2xfgb7v")
-
     # Enable led on boot to notify user of boot status
     # https://gitlab.com/pine64-org/u-boot/-/commit/1a72bd9f5b0d75361b0852ed515d40a47c1e9bfe
     (pine64Patch "1a72bd9f5b0d75361b0852ed515d40a47c1e9bfe" "0z6md01qcsf6xwb6hyn4cgxjz2d0dhv05k5hrqp9kwsh9yghchqq")
```

Comment:
(Side-note) This should be the last and only patch we'll be using compared to mainline U-Boot.